changelogs/unreleased/gh-7523-sigsegv-on-m1-macs.md

```diff
@@ -1,4 +1,4 @@
 ## bugfix/core
 
 * Fixed a bug in fiber switching that could lead to a segmentation fault error
-  on M1/M2 Macs (gh-7523).
+  on AArch64 systems (gh-7523, gh-7985).
```

Comment:
Okay, I was [mislead](https://github.com/tarantool/tarantool/pull/7215#discussion_r910990159) by Nikita.

Reply:
@locker RFC

Reply:
This commit fixes the same issue, it just turned out that fibers are switched in 2 places: coro_transfer and backtrace_collect. So I can add a second changelog, but both of them will contain the same text «Fixed a bug in fiber switching that could lead to a segmentation fault error on AArch64 systems».

Reply:
This is not a bug in fiber switching, but rather a bug in backtrace collection — the fiber switching done here is artificial.

Reply:
Sometimes backtraces are collected on fiber creation and on memory allocation.

Reply:
> Sometimes backtraces are collected on fiber creation and on memory allocation.

Yeah, but it's still a bug in backtrace collection rather than in fiber switching.

I don't want to argue, leaving this up to @locker.

Reply:
Talking solely about changelogs: the goal of the process is to form a full and nice looking release notes for a future release. Second goal: avoid merge conflicts at backporting patches or at least meet it rarely. So I don't see anything negative in editing a changelog entry after its creation if it'll make things better for a reader of the resulting release notes.

Reply:
I'd probably add a new changelog entry, but since there was no release since the changelog entry was committed, I think it's okay to edit the old one.